app/concerns/koudoku/subscription.rb

```diff
@@ -125,6 +125,14 @@ def processing!
             self.stripe_id = customer.id
             self.last_four = customer.sources.retrieve(customer.default_source).last4
 
+            if respond_to? :stripe_subscription_id
+              self.stripe_subscription_id = subscription.id
+            end
+
+            if subscription_owner.respond_to? :stripe_id
+              subscription_owner.update(stripe_id: customer.id)
```

Comment:
@trestrantham Hey. Curious why you'd want to write the Stripe ID to the subscription owner instead of just referencing from the `Subscription` (where it would already be)?

Reply:
This was just a half measure to moving `Stripe::Customer#id` from Subscription to Team while maintaining backward compatibility. Ideally IMO, `Subscription#stripe_id` would be a `Stripe::Subscription#id` and `Team#stripe_id` would be a `Stripe::Customer#id`. I keep getting hung up on the current mismatch. That being said, happy to remove it to avoid confusion. 😄